src/image/loading_displaying.js

```diff
@@ -163,74 +186,48 @@ p5.prototype.loadImage = function(path, successCallback, failureCallback) {
 };
 
 /**
- * Generates a gif of your current animation and downloads it to your computer!
- *
- * The duration argument specifies how many seconds you want to record from your animation.
- * This value is then converted to the necessary number of frames to generate it, depending
- * on the value of units. More on that on the next paragraph.
- *
- * An optional object that can contain two more arguments: delay (number) and units (string).
- *
- * `delay`, specifying how much time we should wait before recording
- *
- * `units`, a string that can be either 'seconds' or 'frames'. By default it's 'seconds'.
+ * Generates a gif from a sketch and downloads it. `saveGif()` may be called
```

Comment:
Is it clear to all potential users that "download" == "save to a local file"?  (Is this clarified somewhere else in the docs?)

Reply:
Good clarification–added!